app/views/episodes/index.html.erb

```diff
@@ -1,14 +1,46 @@
-<p style="color: green"><%= notice %></p>
-
 <h1>Episodes</h1>
 
-<div id="episodes">
-  <% @episodes.each do |episode| %>
-    <%= render episode %>
-    <p>
-      <%= link_to "Show this episode", episode %>
-    </p>
-  <% end %>
+<%= form_with(
+      url: episodes_path,
```

Comment:
I think this needs to vary, depending if you're viewing episodes within a podcast or not?